### PR TITLE
Adding Inventory Number, and Serialized Item Locations and Item Numbers list.

### PR DESCRIPTION
**Pull request by sherriously**

### CODE REVIEW

**Comment by iloveitaly** [comment]
👍 Great work!

Would you mind rebasing the branch against master? Right now I can't merge it :(

**Comment by sherriously** [comment]
I just did a rebase again -- does that work?

File: lib/netsuite/records/serialized_inventory_item_locations.rb [hunk 1]
```diff
@@ -0,0 +1,41 @@
+module NetSuite
+  module Records
+    class SerializedInventoryItemLocations
```
Comment (iloveitaly) [inline lib/netsuite/records/serialized_inventory_item_locations.rb]: @sherriously This doesn't look like it's a valid NetSuite record. Are you trying to implement a list here? lists should use the `Support::Sublist` base class.

Comment (sherriously) [inline lib/netsuite/records/serialized_inventory_item_locations.rb]: I created this based on SerializedInventoryItemLocationsList: http://www.netsuite.com/help/helpcenter/en_US/srbrowser/Browser2015_1/schema/other/serializedinventoryitemlocations.html?mode=package.

Could it be "actions"?

Comment (sherriously) [inline lib/netsuite/records/serialized_inventory_item_locations.rb]: Removed them a few days ago in a later commit!

**Comment by iloveitaly** [comment]
@sherriously I merged these changes with 4516177...97e6aac. Can you take a look and create a new PR for any changes I missed?

Thanks again for the contributions!

**Pull request closed without merging**